dolphinscheduler-extract/dolphinscheduler-extract-base/src/main/java/org/apache/dolphinscheduler/extract/base/server/JdkDynamicServerHandler.java

```diff
@@ -101,6 +104,11 @@ private void processReceived(final Channel channel, final Transporter transporte
                         args = null;
                     } else {
                         args = new Object[standardRpcRequest.getArgs().length];
+                        if (!methodInvoker.isParameterTypeValidated(standardRpcRequest.getArgsTypes())) {
```

Comment:
minor nit: this can happen 1 line earlier before the args assignment

Reply:
Done